repobee_feedback/feedback.py

```diff
@@ -110,11 +112,10 @@ def command(self, api: plug.PlatformAPI):
 
 def _ask_for_open(issue: plug.Issue, repo_name: str, trunc_len: int) -> bool:
     plug.echo(
-        'Processing issue "{}" for {}: {}{}'.format(
+        '\nProcessing issue "{}" for {}:\n{}'.format(
             issue.title,
             repo_name,
-            issue.body[:trunc_len],
-            "[...]" if len(issue.body) > trunc_len else "",
```

Comment:
The `[...]` shouldn't be removed. It gives an indication that there's more to see when the body of the issue is truncated. There's no test for this, which is a bit unfortunate, as it's a key part of the user interface of this plugin.

Reply:
Oooh, how could I have removed this? probably by mistake haha. Sure, I added it back